theme/src/main/assets/page.st

```diff
@@ -46,7 +46,7 @@ $!
     $ elseif (page.properties.("material.author")) $
       <meta name="author" content="$page.properties.("site.author")$">
     $ endif $
-    <link rel="shortcut icon" href="$page.base$$page.properties.("material.favicon")$">
+    <link rel="shortcut icon" href="$page.properties.("assets.hostname")$$page.base$$page.properties.("material.favicon")$">
```

Comment:
I doubt that this is related to https://github.com/apache/incubator-pekko-sbt-paradox/pull/86. I'm not sure if page.properties ("assets.hostname") exists, but using it with $page.base$ may be a bad idea.

Reply:
The icon link of all modules is broken, but the pekko homepage is not, which also proves this. `$page.base$` is the path of a specific project.

Reply:
> The link is to https://pekko.apache.org/docs/pekko-http/current/assets/images/pekko_logo.png
>
> Should be https://pekko.apache.org/assets/images/pekko_logo.png

The previous comments should be no problem. `$page.base$` is not a problem. At present, it is always the root directory of the project, that is.

The documents generated by all these modules (http, connector, pekko) are "assets/xxxx", even if the string template adds `assets.home`, but due to pekko, conn Ector These modules do not have the default value of `assets.home`, so they still do not point the path to CDN.

So I think the PR needs to set a default value for assets.home, or explicitly add the assets.home configuration value for each project.